trunk/3rdparty/signaling/www/demos/js/srs.sdk.js

```diff
@@ -392,19 +392,19 @@ function SrsRtcPlayerAsync() {
                 .replace("rtc://", "http://");
 
             var vhost = a.hostname;
-            var app = a.pathname.substr(1, a.pathname.lastIndexOf("/") - 1);
-            var stream = a.pathname.substr(a.pathname.lastIndexOf("/") + 1);
+            var app = a.pathname.substring(1, a.pathname.lastIndexOf("/") - 1);
```

Comment:
There is a bug:

```
'1234567890'.substr(1,3)
'234'

'1234567890'.substring(1,3)
'23'
```

The two are not equal.

Reply:
You are absolutely right. Not sure why but I clearly didn't pay enough attention there.
I have now rechecked the changes and fixed the issues